src/States/MainMenuState.cpp

```diff
@@ -215,7 +215,7 @@ void MainMenuState::onFadeComplete()
 void MainMenuState::btnNewGameClicked()
 {
 	if (mFileIoDialog.visible()) { return; }
-	if(dlgOptions.visible()) { return; }
+	if (dlgOptions.visible()) { return; }
```

Comment:
Despite the updates, I believe some still exist in this PR:
```
grep -rn "if(" src | wc -l
```
On `master`: 38
On this branch: 48

Reply:
Oh, sorry. That commit was just for this file.

Reply:
What I meant was there are still 10 new `if(` blocks introduced on this branch.